package.json

```diff
@@ -75,11 +75,11 @@
     "codelyzer": "^6.0.0",
     "eslint": "^6.8.0",
     "eslint-plugin-angular": "^4.0.1",
-    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsdoc": "^32.2.0",
     "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "^5.1.1",
+    "karma": "^6.3.2",
```

Comment:
When I installed the new version of DSP-UI, `npm i` also updated other packages (because of security issues).